versioned/storage/cache/src/main/java/org/projectnessie/versioned/storage/cache/CaffeineCacheBackend.java

```diff
@@ -58,15 +60,15 @@ class CaffeineCacheBackend implements CacheBackend {
     refCacheTtlNanos = config.referenceTtl().orElse(Duration.ZERO).toNanos();
     refCacheNegativeTtlNanos = config.referenceNegativeTtl().orElse(Duration.ZERO).toNanos();
 
-    Caffeine<CacheKeyValue, byte[]> cacheBuilder =
+    Caffeine<CacheKeyValue, CacheKeyValue> cacheBuilder =
```

Comment:
nit: why not use two fixed `String` objects for values: one for "live" entries, another for `NON_EXISTING_SENTINEL`?.. probably not a lot of difference memory-wise, but it could be more readable, I hope.

Reply:
Would need a composite for the serialized and soft-reference anyway. Sure, it's a bit more readable, but I'm a bit more concerned about heap pressure here.

Reply:
Values could we static constants. We only need them to distinguish the sentinel from real values, but the actually value is always in the key... anyway this is totally optional.

Reply:
I think I was confused. Please disregard.